UnityProject/Assets/Scripts/Player/Huggable.cs

```diff
@@ -82,8 +96,46 @@ private bool TryFieryHug()
 		return false;
 	}
 
+	private void Judgement(LivingHealthMasterBase puller)
+	{
+		if (lastPuller == null || lastPuller != puller)
+		{
+			JudgementCooldown(puller);
+			return;
+		}
+		if (DMMath.Prob(tailPullJudgementChance))
+		{
+			puller.Gib();
```

Comment:
I think it important to communicate to the performer and observers why they just got gibbed. Something witty.